dashboard/public/500.html

```diff
@@ -1,54 +1,101 @@
-    background-color: white;
-    padding: 7px 4em 0 4em;
+    font-family: 'Gotham 4r', sans-serif;
+    font-size: 14px;
+    line-height: 22px;
+    font-weight: normal;
+    color: dimgray;
+    margin: 0;
   }
 
   h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
+    color: #7665a0;
+    font-size: 28px;
+    font-family: 'Gotham 7r', sans-serif;
+    margin-top: 50px;
+    margin-bottom: 30px;
+    line-height: 1.2em;
   }
 
-  body > p {
-    width: 33em;
-    margin: 0 auto 1em;
-    padding: 1em 0;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-bottom-color: #999;
+  button {
+    -webkit-appearance: none;
+    -webkit-user-select: none;
+    -webkit-writing-mode: horizontal-tb;
+    align-items: flex-start;
+    background-color: #ffa400;
+    border-color: #ffa400;
+    background-image: none;
     border-bottom-left-radius: 4px;
     border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow:0 3px 8px rgba(50, 50, 50, 0.17);
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-image-outset: 0px;
+    border-image-repeat: stretch;
+    border-image-slice: 100%;
+    border-image-source: none;
+    border-image-width: 1;
+    border-left-style: solid;
+    border-left-width: 1px;
+    border-right-style: solid;
+    border-right-width: 1px;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    border-top-style: solid;
+    border-top-width: 1px;
+    box-sizing: border-box;
+    color: #fff;
+    cursor: pointer;
+    display: inline-block;
+    font-family: 'Gotham 4r', sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    height: 34px;
+    letter-spacing: normal;
+    line-height: 20px;
+    margin-bottom: 0px;
+    margin-left: 0px;
+    margin-right: 0px;
+    margin-top: 30px;
+    padding-bottom: 6px;
+    padding-left: 12px;
+    padding-right: 12px;
+    padding-top: 6px;
+    text-align: center;
+    text-indent: 0px;
+    text-shadow: none;
+    text-transform: none;
+    vertical-align: middle;
+    white-space: nowrap;
+    word-spacing: 0px;
+    writing-mode: lr-tb;
+  }
+
+  .error-image {
+    margin-top: 50px;
+  }
+
+  .error-parent {
+    text-align: center;
+  }
+
+  .error-child {
+    display: inline-block;
+    max-width: 300px;
   }
   </style>
 </head>
 
 <body>
-  <!-- This file lives in public/500.html -->
-  <img src="/sitedown.jpg" alt="The site is down. Please check back shortly. We're working on it"/>
+  <div class="error-parent">
+    <div class="error-child">
+      <img class="error-image" src="/shared/images/sad-bee-avatar.png" alt="Page not found"/>
+      <h1>500: Server error</h1>
+      <div>Sorry, we're having some technical difficulties. Please try refreshing this page again later. If the problem persists, you can contact us at <a href="mailto:support@code.org">support@code.org</a>.</div>
```

Comment:
Any chance we're worried about this email address being scraped?  Would we rather link to our support site?

Reply:
Also it might be worth wrapping this long line.